README.md

```diff
@@ -2,26 +2,26 @@
 
 <img src="https://user-images.githubusercontent.com/69774407/119630519-b11e2480-be17-11eb-8f89-f1179db1948e.png" width="100" height="100">
 
-## The Pomodoro Technique is a time management method.
-#### The technique uses a timer to break down work into intervals, traditionally 25 minutes in length, separated by short breaks. Each interval is known as a pomodoro, from the Italian word for 'tomato', after the tomato-shaped kitchen timer that Cirillo used as a university student.
+#### This technique helps in managing working time as it divides it into specific periods, each period is called pomodoro, where the length of the pomodoro is 25 minutes, and after every 25 minutes there is a 5-minute break, and if four pomodoro cycles are completed, there will be a 15-minute break.
```

Comment:
Change this line and the line you delete it to be regular paragraph text instead of a header text.